scripts/standalone.mk

```diff
@@ -37,7 +37,16 @@ ifeq ($(RISCV_CMODEL),)
 RISCV_CMODEL = medany
 endif
 
-ifeq ($(LINK_TARGET),)
+ifeq ($(PROGRAM),dhrystone)
+LINK_TARGET = ramrodata
+# Ground rules
+RISCV_XCFLAGS += -mexplicit-relocs -fno-inline
+# With inline and without lto
+#RISCV_XCFLAGS += -mexplicit-relocs -finline
+# Best Score
+#RISCV_XCFLAGS += -finline -flto -fwhole-program
+RISCV_XCFLAGS += -DDHRY_ITERS=$(TARGET_DHRY_ITERS)
```

Comment:
Why are some of these flags commented out?

Also, this should be split into two separate conditionals. The LINK_TARGET should remain here, and the RISCV_XCFLAGS should go below in the Software Flags section.

Reply:
The terms to use for the three options should be
Legal, Fast and Max Performance
But we should provide a make flag just like LINK_TARGET for user to select which one for their build.
The default should be Legal.

Reply:
@nategraff-sifive We want to provide the scores as ARM did.
https://developer.arm.com/ip-products/processors/cortex-m/cortex-m0

Reply:
That's why I put 3 options on it, I will add make flag for this~

Reply:
I've added a make flag "DHRY_OPTION"  for this and move these to Software Flags sections.